stamp.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Use the first tag that points to the current HEAD
+# if no tag is found, the latest git commit is used as a fallback
+
+CURRENT_TAG=$(git tag -l --points-at HEAD | head -n1);
```

Comment:
Can we remove the trailing semicolons here?

Reply:
Done!